src/Objs/Redirect/RedirectComponent.tsx

```diff
@@ -18,21 +18,17 @@ class RedirectComponent extends React.Component {
         window.location.replace(url)
       }
     })
-  }
-
-  render() {
-    const link = this.props.page.get('link')
+  }, [page])
```

Comment:
* [ ] FYI we could move the assignment to `link` up and use `link` for the dependency instead of `page`. Maybe even use `url` as it best explains the actual effect to the reader.

Reply:
I moved the assignments up. Now we have 3 dependencies instead. If you have ideas for improvement, let me know 🙂